chanfitness/chaneventstore_test.go

```diff
@@ -93,10 +94,15 @@ func TestMonitorChannelEvents(t *testing.T) {
 		t.Fatalf("Could not create vertex: %v", err)
 	}
 
-	shortID := lnwire.ShortChannelID{
-		BlockHeight: 1234,
-		TxIndex:     2,
-		TxPosition:  2,
+	// Create test outpoint.
+	var txid [32]byte
+	if _, err := rand.Read(txid[:]); err != nil {
+		t.Fatalf("error creating txid: %v", err)
+	}
```

Comment:
Would pull the later refactor to create a test channel to this commit, to prevent touch this code twice.

Reply:
nice, will do!